apps/_pomodoro_app/realtime/application/utils/createRandomWord.js

```diff
@@ -0,0 +1,18 @@
+function createRandomWord(length) {
```

Comment:
Usually, I'd like to create a file with different utils. for example: `string.utils.js` or `random.utils.js` and put more than one function inside. It's not so common to use one file for just one function.